### PR TITLE
VZ-4141

### PR DESCRIPTION
**Pull request by shashankkumarIITB**
# Description

Added changes to enable `kubectl port-forwarding` with Istio >= 1.10 for testing examples.

Fixes VZ-4141

# Checklist 

As the author of this PR, I have:

- [x] Checked that I included or updated copyright and license notices in all files that I altered
- [x] Added or updated unit tests for any new functions I added
- [x] Added or updated integration tests if appropriate
- [x] Added or updated acceptance tests if appropriate

Code reviewer, please confirm this PR:

- [ ] Addressed the requirement and meets the acceptance criteria
- [ ] Does not introduce unrelated or spurious changes
- [ ] Does not introduce any unapproved dependency
- [ ] Makes sense and it easy to understand, and/or difficult areas of code are clearly documented so that they can be understood

### CODE REVIEW

File: examples/todo-list/todo-list-components.yaml [hunk 1]
```diff
@@ -16,6 +16,8 @@ spec:
           name: todo-domain
           namespace: todo-list
         spec:
+          adminServer:
```
Comment (mgianatagh) [inline examples/todo-list/todo-list-components.yaml:19]: This change should also be made to the multicluster example: 

examples/multicluster/todo-list/todo-list-components.yaml

Comment (shashankkumarIITB) [reply examples/todo-list/todo-list-components.yaml:19]: I was going to ask that in the standup. Thanks!

**Review by mgianatagh — DISMISSED** [review]

**Review by markxnelson — APPROVED** [review]

**Review by bobdonat — APPROVED** [review]

File: examples/bobs-books/bobs-books-comp.yaml [hunk 2]
```diff
@@ -187,6 +187,8 @@ spec:
             weblogic.resourceVersion: domain-v8
             weblogic.domainUID: bobbys-front-end
         spec:
+          adminServer:
+            adminChannelPortForwardingEnabled: true
```
Comment (ayaranga) [inline examples/bobs-books/bobs-books-comp.yaml:191]: Don't we need to add this change to bobs-bookstore as well? Why is the issue not seen with Verrazzano 1.0.* versions?

Comment (shashankkumarIITB) [reply examples/bobs-books/bobs-books-comp.yaml:191]: I will look into it.

File: examples/bobs-books/bobs-books-app.yaml [hunk 3]
```diff
@@ -39,6 +39,8 @@ spec:
                 - paths:
                     - path: "/bobbys-front-end"
                       pathType: Prefix
+                  destination:
```
Comment (bobdonat) [inline examples/bobs-books/bobs-books-app.yaml:42]: Why does the destination port now need to be specified?  This was not the case in the past.

Comment (shashankkumarIITB) [reply examples/bobs-books/bobs-books-app.yaml:42]: Asha has put up a question in Verrazzano-dev. We are yet to receive an answer for that. This is a temporary fix for now.  If we get a go ahead on the question, I have created a separate PR here - https://github.com/verrazzano/verrazzano/pull/2083

**Review by markxnelson — APPROVED** [review]

**Pull request merged**